app/templates/components/item-component.hbs

```diff
@@ -1,9 +1,9 @@
 <div>
   <a href="{{item.externalUrl}}">{{item.title}}</a>
   <div>
-    <span>{{item.points}} points by {{item.user}} {{item.timeAgo}} </span>
-    <span {{action markAsRead}} class='hide-button test__item-delete-button'>
```

Comment:
This will be confusing during the testing workshop so I'm keen to remove it (it was unused).